beamer/agent/events.py

```diff
@@ -9,7 +9,8 @@
 from hexbytes import HexBytes
 from requests.exceptions import HTTPError, ReadTimeout, RequestException
 from web3 import HTTPProvider, Web3
-from web3.contract import Contract, get_event_data
+from web3._utils.events import get_event_data
```

Comment:
Isn't there another place where we could import `get_event_data` from?
In general, we should not use private things (modules, items) like that.